Drv/LinuxGpioDriver/LinuxGpioDriver.fpp

```diff
@@ -18,15 +12,20 @@ module Drv {
 
     text event port LogText
 
-    sync input port gpioRead: Drv.GpioRead
-
     time get port Time
 
     # ----------------------------------------------------------------------
     # Events
     # ----------------------------------------------------------------------
+    event OpenChip(chip: string, label: string, pins: U32) severity diagnostic format "Opened {}[{}] with {} lines"
```

Comment:
Could we make those format strings a little more verbose for users to be able to google / figure out things on their own easily?

Reply:
Like `Failed to open pin {} on chip {} ({})`